server/index.js

```diff
@@ -1,24 +1,34 @@
 require('dotenv').config();
 require('../db/index.js');
+const path = require('path');
 const express = require('express');
 const Reviews = require('../models/Reviews');
 const Products = require('../models/Products');
 
 const app = express();
 
-app.use(express.static(__dirname + '/../client/dist'));
+app.use(express.static(path.join(__dirname, '/../client/dist')));
+
+const calculateRatings = (reviews) => {
+  const { ratings } = reviews;
+  console.log('ratings', ratings);
+  const { overall, quality, value, comfort, sizing, style } = ratings;
+  console.log('categories', overall, quality, value, comfort, sizing, style);
+  return ratings;
+};
 
 app.get('/', (req, res) => res.send('Hello World!'));
```

Comment:
Is this needed?  A GET request at the root should send the index.html file in your static folder.

Reply:
this function isnt implemented yet - i was setting up server tests for its initial work.